frontend/src/admin/createUser/CreateUserPage.jsx

```diff
@@ -6,7 +6,6 @@ import CreateUserPageContents from "./CreateUserPageContents";
 export default function CreateUserPage() {
   const breadcrumbs = [
     ["Home", "/Home"],
-    ["Admin", "/Admin"],
```

Comment:
If we can have breadcrumbs without links I would keep this breadcrumb. I feel like it emphasizes that this is an admin action.

Reply:
4ec1a5b

Reply:
Link will still be hoverable but it won't redirect to anything.